### PR TITLE
Add URL check/validation to PRs

### PR DESCRIPTION
**Pull request by ScriptAutomate**
Makes life easier, checks validity of all URLs. This will help validate that a URL actually works when a PR is submitted, but it will also validate ALL URLs work, which will allow for catching broken URLs in a more automated fashion.

- Added Linode to `.brokignore`, as the URL works, but `brok` gives error around it
- Fixed Facebook page URL

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -9,4 +9,4 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - run: npx awesome-lint
+      - run: npx awesome-lint
```
Comment (hbokh) [inline .github/workflows/main.yml]: Please add a newline at the end of the file, as suggested.

File: .github/workflows/url-check.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        path: .brokdb
+        key: ${{ hashFiles('.github/workflows/main.yml') }}-${{ hashFiles('.brokignore') }}-${{ steps.get-date.outputs.date }}
+
+    - name: Install brok
+      run: |
+        wget https://github.com/smallhadroncollider/brok/releases/download/1.1.0/brok-1.1.0_x86-64-linux.deb
+        dpkg -i brok-1.1.0_x86-64-linux.deb
+    - name: Run brok
+      run: |
+        brok --ignore $(cat .brokignore) README.md > /dev/null
```
Comment (hbokh) [inline .github/workflows/url-check.yml]: Add newline to EOF.

File: .github/workflows/main.yml (hunk 1 above)
Comment (ScriptAutomate) [inline .github/workflows/main.yml]:
```suggestion
      - run: npx awesome-lint

```

File: .github/workflows/url-check.yml (hunk 2 above)
Comment (ScriptAutomate) [inline .github/workflows/url-check.yml]:
```suggestion
        brok --ignore $(cat .brokignore) README.md > /dev/null

```

**Review by hbokh — APPROVED** [review]
Thanks!

**Pull request merged**